Desktop/widgets/listmodelfiltereddataentry.cpp

```diff
@@ -190,9 +208,12 @@ void ListModelFilteredDataEntry::initTableTerms(const TableTerms& terms)
 	for (int rowIndex : _tableTerms.rowIndices)
 	{
 		size_t row = static_cast<size_t>(rowIndex) - 1;
-
-		_enteredValues[row] = _tableTerms.values[0][valIndex++].toDouble();
+		if (_tableTerms.values[0].size() < valIndex)
+			_enteredValues[row] = _tableTerms.values[0][valIndex].toDouble();
+		else
+			_tableTerms.values[0].push_back(_tableView->defaultValue());
```

Comment:
Which @boutinb does here! 
(Getting a defaultValue that is, @koenderks have you tried setting that value?)

Reply:
No

Reply:
It does not seem to know it
![image](https://user-images.githubusercontent.com/25059399/109482171-f8498300-7a7d-11eb-8d0e-d70061e30315.png)

Reply:
The property is defaultEmptyValue: "0". I have just checked and it works.

Reply:
I should make the defaultEmptyValue property a Variant, so that you can write defaultEmptyValue: 0 (without quote).